### PR TITLE
[plugin.video.putio] 3.0.0

### PR DESCRIPTION
**Pull request by BatuAksoy**
### Description
This commit migrates existing putio add-on to matrix. See old add-on here: https://github.com/xbmc/repo-plugins/tree/helix/plugin.video.putio
- Updated xbmc.python to 3.0.0
- Removed/Updated python2 specific codes
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.putio/addon.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.service" library="service.py" />
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">Put.io for Kodi</summary>
+        <description lang="en">Put.io is a service that combines cloud storage and bittorrent. You need an account to use this plug-in. Get one from https://put.io</description>
+        <disclaimer lang="en_GB">Please read terms of service at https://put.io/terms-of-service/</disclaimer>
+        <language>en tr</language>
+        <platform>all</platform>
+        <license>GNU GENERAL PUBLIC LICENSE Version 3, 29 June 2007</license>
```
Comment (enen92) [inline plugin.video.putio/addon.xml]: GPL-3.0-only or GPL-3.0-or-later instead of the full name

File: plugin.video.putio/addon.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        <forum></forum>
```
Comment (enen92) [inline plugin.video.putio/addon.xml]: Drop if empty

File: plugin.video.putio/main.py [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+            continue
+
+        li.setInfo(type=item_type, infoLabels=info_labels)
+
+        context_menu_items = [(I18N(32040), 'Container.Refresh')]
+        if not item.is_shared:
+            # url when a delete action is triggered
+            delete_ctx_url = build_url(action='delete', item=item.id)
+            context_menu_items.append(
+                (I18N(32042), 'XBMC.RunPlugin(%s)' % delete_ctx_url))  # delete context
```
Comment (enen92) [inline plugin.video.putio/main.py]: Drop xbmc prefix, no longer valid

File: plugin.video.putio/resources/__init__.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```
Comment (enen92) [inline plugin.video.putio/resources/__init__.py]: Shouldn't be needed anymore in python3

File: plugin.video.putio/resources/lib/__init__.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```
Comment (enen92) [inline plugin.video.putio/resources/lib/__init__.py]: Same

**Review by enen92 — CHANGES_REQUESTED** [review]
See comments

**Comment by BatuAksoy** [comment]
Thank you for your review and feedbacks @enen92.
Now it should be okay

**Review by enen92 — APPROVED** [review]

**Comment by enen92** [comment]
Thanks for adjusting

**Pull request merged**